src/map/npc.c

```diff
@@ -3724,7 +3724,7 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 		return strchr(start,'\n');// skip and continue
```

Comment:
Event names should only be validated at run time, just like any other script. You would have to change the load order to make all the monster scripts go last to validate the event names.